gradle/scripts/lib/java.gradle

```diff
@@ -125,6 +160,11 @@ configure(projectsWithFlags('java')) {
                 languageVersion = JavaLanguageVersion.of(rootProject.ext.testJavaVersion)
             }
         }
+
+        // disable tests for projects which target a higher java version
+        if (testJavaVersion < targetJavaVersion) {
```

Comment:
nit: For consistency: `rootProject.ext.testJavaVersion` 😄

Reply:
Both done 😄